### PR TITLE
Fix a few typos

### PR DESCRIPTION
**Pull request by Blake-Madden**
Misspellings, article mismatches

### CODE REVIEW

**Comment by Jay-o-Way** [comment]
LGTM

File: hub/apps/design/controls/contact-card.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ The contact card displays contact information, such as the name, phone number, a
 > **Important APIs**: [ShowContactCard method](/uwp/api/windows.applicationmodel.contacts.contactmanager.showcontactcard),   [ShowFullContactCard method](/uwp/api/windows.applicationmodel.contacts.contactmanager.showfullcontactcard),  [IsShowContactCardSupported method](/uwp/api/windows.applicationmodel.contacts.contactmanager.IsShowContactCardSupported),  [Contact class](/uwp/api/Windows.ApplicationModel.Contacts.Contact)  
 
 There are two ways to display the contact card:  
-* As a standard contact card that appears in a flyout that is light-dismissable--the contact card dissapears when the user clicks outside of it. 
+* As a standard contact card that appears in a flyout that is light-dismissable--the contact card disapears when the user clicks outside of it.
```
Comment (Jay-o-Way) [inline hub/apps/design/controls/contact-card.md]: double P please

**Review by Jay-o-Way — CHANGES_REQUESTED** [review]
Great job! just _one_ little thing :)

File: hub/apps/design/controls/contact-card.md (hunk 1 above)
Comment (Blake-Madden) [reply hub/apps/design/controls/contact-card.md]: Got it.

**Comment by GrantMeStrength** [comment]
Wow thanks for fixing all these!

**Pull request merged**